packages/l/libsdl_gfx/xmake.lua

```diff
@@ -1,14 +1,108 @@
 package("libsdl_gfx")
+    add_deps("libsdl")
+    on_load(function(package)
+        package:add("includedirs", "include")
+    end)
 
     set_homepage("https://www.ferzkopp.net/wordpress/2016/01/02/sdl_gfx-sdl2_gfx/")
     set_description("Simple DirectMedia Layer primitives drawing library")
 
```

Comment:
Please put on_load/add_deps on top of on_install, and put set_homepage/set_urls on top.